pkg/collectors/cluster_resource_quota.go

```diff
@@ -77,6 +77,72 @@ var (
 				return f
 			}),
 		},
+		metric.FamilyGenerator{
+			Name: "openshift_clusterresourcequota_ns_usage",
```

Comment:
We usually use the full name instead, could you update the "openshift_clusterresourcequota_namespace_usage" ?

Reply:
done